src/app/checkout/step-2/step-2.component.js

```diff
@@ -50,7 +52,9 @@ class Step2Controller {
   }
 
   handlePaymentChange (selectedPaymentMethod) {
-    this.defaultPaymentType = selectedPaymentMethod['account-type'] ? selectedPaymentMethod['account-type'] : selectedPaymentMethod['card-type']
+    const paymentType = selectedPaymentMethod['account-type'] ? selectedPaymentMethod['account-type'] : selectedPaymentMethod['card-type']
```

Comment:
I'm unsure of the JS stadards in this document, but a simplified way to do this is the below.
```suggestion
const paymentType = selectedPaymentMethod['account-type'] ||  selectedPaymentMethod['card-type']